debs/bionic/archivematica/Makefile

```diff
@@ -7,7 +7,7 @@ GPG_ID ?= 0F4A4D31
 PACKBUILD_EXTRA_ARGS ?= 
 #PACKBUILD_EXTRA_ARGS ?= "-b 1"
 BRANCH     ?= qa/1.x
-VERSION     ?= 1.8.0
+VERSION     ?= 1.8.1
```

Comment:
If this is being changed before the 1.8.1 release, I think it should go on it's own PR